frontends/aiger/aigerparse.cc:307:24: error: expected ')' before 'PRId64'

## Steps to reproduce the issue

 1) Checkout Yosys at ac2fc3a144fe1094bedcc6b3fda8a498ad43ae76
 2) Try to build with gcc/g++ version 7.3.0
 3) Build fails with
    ```
    frontends/aiger/aigerparse.cc:307:24: error: expected ')' before 'PRId64'
       log_error("Offset %" PRId64 ": unable to read literal!\n", static_cast<int64_t>(f.tellg()));
    ```

## Expected behavior

Build completes.

## Actual behavior

The exact compile command was;
```
x86_64-conda_cos6-linux-gnu-gcc -o frontends/aiger/aigerparse.o -c -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -I$PREFIX -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/yosys-0.8_2385_gac2fc3a1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -Wall -Wextra -ggdb -I. -I"./" -MD -D_YOSYS_ -fPIC -I$PREFIX/include -std=c++11 -Os -I$PREFIX/include -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -I$PREFIX/include -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER frontends/aiger/aigerparse.cc
```

```
frontends/aiger/aigerparse.cc: In function 'uint32_t Yosys::parse_xaiger_literal(std::istream&)':
frontends/aiger/aigerparse.cc:307:24: error: expected ')' before 'PRId64'
   log_error("Offset %" PRId64 ": unable to read literal!\n", static_cast<int64_t>(f.tellg()));
                        ^~~~~~
frontends/aiger/aigerparse.cc:307:93: warning: spurious trailing '%' in format [-Wformat=]
   log_error("Offset %" PRId64 ": unable to read literal!\n", static_cast<int64_t>(f.tellg()));
```

`version: 0.8_2385_gac2fc3a1` -- https://github.com/YosysHQ/yosys/commit/ac2fc3a144fe1094bedcc6b3fda8a498ad43ae76

Full build log avaliable at https://travis-ci.org/mithro/litex-conda/jobs/569494123

It is unclear to me why PRId64 is missing. Looking at the `intypes.h` file it has;
```cpp
/* The ISO C99 standard specifies that these macros must only be
   defined if explicitly requested.  */
#if !defined __cplusplus || defined __STDC_FORMAT_MACROS
```

https://www.gnu.org/software/gnulib/manual/html_node/inttypes_002eh.html
> On some hosts that predate C++11, when using C++ one must define __STDC_FORMAT_MACROS to make visible the declarations of format macros such as PRIdMAX.

The following patch seems to make it compile;
```patch
diff --git a/frontends/aiger/aigerparse.cc b/frontends/aiger/aigerparse.cc
index 85ee34e2..bd799846 100644
--- a/frontends/aiger/aigerparse.cc
+++ b/frontends/aiger/aigerparse.cc
@@ -30,6 +30,7 @@
 #include <libkern/OSByteOrder.h>
 #define __builtin_bswap32 OSSwapInt32
 #endif
+#define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
 #include "kernel/yosys.h"
```

The script in `libs/minisat/00_UPDATE.sh` seems to add the `__STDC_FORMAT_MACROS` define to all of the files?

--- a/frontends/aiger/aigerparse.cc
+++ b/frontends/aiger/aigerparse.cc
@@ -30,6 +30,7 @@
 #include <libkern/OSByteOrder.h>
 #define __builtin_bswap32 OSSwapInt32
 #endif
+#define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
 #include "kernel/yosys.h"

